.changeset/clean-geese-cheat.md

```diff
@@ -0,0 +1,5 @@
+---
+'@withease/web-api': patch
+---
+
```

Comment:
🚨  I do not see any updates in `website/docs`. In my opinion, it's required to update docs before release. So, it's a blocker.

Reply:
[Updated](https://github.com/igorkamyshev/withease/pull/70/commits/f3f7563189d677fa9e36aba75091416780c49254) the documentation and demo to contain information and samples of new stores